### ISSUE TITLE
bgmi search only works with source bangumi_moe

### ISSUE THREAD

**Issue by oshino29** [codysk/bgmi-docker-all-in-one#3]
when using `bgmi search` command(which is the way for bgmi to download episodes outside of currently bangumi calendar) with mikan_project or dmhy, this error will be raised up:
```
[x] 'dict' object has no attribute 'title'
```
how to reproduce this:
```
#create a new container
docker run --name="bgmi_test" -p 80:80 -p 9091:9091 codysk/bgmi-all-in-one
#connect to the bash
docker exec -it bgmi_test bash
#try bgmi seaerch command with different souces
bgmi source mikan_project
bgmi search "佐贺偶像是传奇"
bgmi source dmhy
bgmi search "佐贺偶像是传奇"
bgmi source bangumi_moe
bgmi search "佐贺偶像是传奇"
```
error will be raised up when using dmhy or mikan_project as source

(The official bgmi binary from pip works perfect with all souces so I think this issue should be listed here)

**Comment by codysk** [comment]
This issue cause by bgmi mainline source, which is not publish to pip yet.

And here is a PR about this issue.
https://github.com/BGmi/BGmi/pull/244

In the meantime, you can use previous version `codysk/bgmi-all-in-one:1.1.4` avoid the search issue.

**Comment by codysk** [comment]
Fix on [1.1.7](https://github.com/codysk/bgmi-docker-all-in-one/releases/tag/1.1.7)
latest tag also updated.
